module_1_task_20/task_4.py

```diff
@@ -0,0 +1,8 @@
+def multiply_decorator(fac):
+    return lambda f: lambda: f() * fac
```

Comment:
Nice! But remember that under the hood it would look like this:
```suggestion
def multiply_decorator(fac):
    def decorator(f):
        def wrapper():
            return f() * fac
        return wrapper
    return decorator
```